packages/fossil/autosetup-cc.tcl.patch

```diff
@@ -0,0 +1,29 @@
+--- ./autosetup/cc.tcl	2021-07-02 18:16:01.000000000 +0530
++++ ./autosetup/cc.tcl.mod	2021-08-03 07:04:16.933978205 +0530
+@@ -109,25 +109,7 @@
+ #
+ # Checks that the given include files can be used.
+ proc cc-check-includes {args} {
+-	cc-check-some-feature $args {
```

Comment:
Why did you need to disable this?

Reply:
The checks are probably broken. They don't work on device builds. I didn't test in the CI. Trying it now